Pages/Crime.cs

```diff
@@ -24,9 +24,12 @@ public partial class Crime
         public string IncidentNo { get; set; }
 
         [JsonProperty("date_reported")]
+        [DataType(DataType.Date)]
```

Comment:
Added "[DateTime(DateType.Date)]" syntax for displaying the Date and Time and for that I added "ComponentMode1.DateAnnotations" in Index.cshtml